schemas/oic.sec.roles.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://www.openconnectivity.org/ocf-apis/security/schemas/oic.sec.roles.json#",
+  "title": "Device Roles information",
+  "definitions": {
+    "oic.sec.cred": {
+      "type": "object",
+      "properties": {
+        "$ref": "oic.sec.cred-base.json#definitions/oic.sec.cred"
+      },
+      "required": ["credid", "subjectuuid", "credtype", "publicdata" ]
```

Comment:
I don't think "subjectuuid" should be required for oic.sec.roles (this is the set of required Properties for a Retrieve response).  IoTivity does include "subjectuuid" so it won't break IoTivity cert, but it's not used in /roles and could be omitted by a different implementation.

Reply:
This is the only different change in PR80 w.r.t. PR#81 
(https://github.com/openconnectivityfoundation/security-models/pull/80/commits/0c6384fd635b54a9c0d25276ed4f53e91371bf36).
I will take this change too, and put that on top of current PR#81

Reply:
+ Is there any change in decision on the DELETE with query parameter part (current PR does not have DELETE with query parameter, but we decided to include that back) ?